analyzers/src/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)\..\Analyzers.targets" />
```

Comment:
this makes sense to be a `Directory.Build.targets` because it's importing the other targets